ONSdigital/eq-author-app
# Added validation errors to calculated summary page

### What is the context of this PR?
Validation errors were not being displayed on calculated summary page

### How to review 
1. Create calculated summary page
2. Ensure validation error & navigation menu item error indicator appears for the title field
